src/widgets/VaccineChart.js

```diff
@@ -34,112 +42,214 @@ export class VaccineChart extends React.Component {
+
+    if (!(breaches.length > 0)) return null;
+
+    return breaches.map(breach => (
+      <VictoryScatter
+        dataComponent={<HazardPoint onPress={onPress} breach={breach} />}
+        data={breach}
+        {...dataKeys}
+      />
+    ));
+  }
+
+  // Helper method for rendering vaccine chart.
+  renderPlot() {
+    const { minLine, maxLine } = this.props;
 
     if (minLine.length + maxLine.length === 0) return null;
 
-    const dataKeys = { x: 'date', y: 'temp' };
+    const { width, height } = this.state;
 
-    const lines = [...minLine, ...maxLine];
+    // First render will always return null due to sizing using onLayout.
+    if (!width || !height) return null;
 
-    const minDomain =
-      lines.reduce((reducedTemp, { temp }) => Math.min(reducedTemp, temp), minTemp) - 1;
-    const maxDomain =
-      lines.reduce((reducedTemp, { temp }) => Math.max(reducedTemp, temp), maxTemp) + 1;
-
-    const minTempPlotLine =
-      // minTemp default prop is Infinity.
-      minTemp !== Infinity ? (
-        <VictoryLine
-          y={() => minTemp}
-          style={{
-            data: { strokeDasharray: minLineStyles.strokeDasharray, stroke: minLineStyles.stroke },
-          }}
-        />
-      ) : null;
-
-    const maxTempPlotLine =
-      // maxTemp default prop is -Infinity.
-      maxTemp !== -Infinity ? (
-        <VictoryLine
-          y={() => maxTemp}
-          style={{
-            data: { strokeDasharray: maxLineStyles.strokeDasharray, stroke: maxLineStyles.stroke },
-          }}
-        />
-      ) : null;
-
-    const minLinePlotLine =
-      minLine.length > 0 ? (
-        <VictoryLine
-          data={minLine}
-          interpolation={minLineStyles.interpolation}
-          style={{ data: { stroke: minLineStyles.stroke } }}
-          {...dataKeys}
-        />
-      ) : null;
-
-    const minLinePlotScatter =
-      minLine.length > 0 ? (
-        <VictoryScatter
-          data={minLine}
-          style={{ data: { fill: minLineStyles.fill } }}
-          {...dataKeys}
-        />
-      ) : null;
-
-    const maxLinePlotLine =
-      maxLine.length > 0 ? (
-        <VictoryLine
-          data={maxLine}
-          interpolation={maxLineStyles.interpolation}
-          style={{ data: { stroke: maxLineStyles.stroke } }}
-          {...dataKeys}
-        />
-      ) : null;
-
-    const maxLinePlotScatter =
-      maxLine.length > 0 ? (
-        <VictoryScatter
-          data={maxLine}
-          style={{ data: { fill: maxLineStyles.fill } }}
-          {...dataKeys}
-        />
-      ) : null;
-
-    const hazardPlotScatter =
-      hazards.length > 0 ? (
-        <VictoryScatter dataComponent={<HazardPoint />} data={hazards} {...dataKeys} />
-      ) : null;
+    const { minDomain, maxDomain } = this.getDomain();
 
     return (
       <Svg>
         <VictoryChart
           width={width}
           height={height}
+          padding={{ top: 20, left: 50, right: 50, bottom: 50 }}
```

Comment:
change not required
you can have chartStyles as
```
const chartStyles = {
  padding: ...,
  styles: ...
}
```

And then here can just do 
{...chartStyles}

Reply:
Fixed by 4a915d3.